### DISCUSSION TITLE
Handling 8khz audio originating from a phone

### DISCUSSION THREAD

**Discussion by bowen-facere** [deepgram/community#973] [General help]
Hi all,

We've been using nova-2 for transcription over audio that originates from a phone line, for example audio originally in g.729 codec. We have noticed that there are scenarios where what is said is perceivable by human ears but results in varied inaccuracies in the final transcript.

Does anyone have any suggestions?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by deepgram-community** [comment]
Based on the feedback and discussions from other users, there are a few suggestions that might help improve the accuracy of your transcriptions:

1. Try using the `Nova-2-phonecall` model: This model is specifically designed for phone call accuracy and might perform better with your audio data.

2. Adjust the audio level: Deepgram seems to be sensitive to audio levels. If the audio level is too low, it might affect the quality of the transcriptions. You might want to consider adjusting the audio level before feeding it into the model.

3. Use noise suppression: If there's a lot of background noise in your audio data, it might be affecting the transcription quality. You could try using a noise suppression tool to reduce the background noise.

4. Check your request format: Some users have reported a decrease in error rate after removing the encoding value and adding a sample rate to their requests.

5. Try the `enhanced` model: If the speaker in your audio data is far from the microphone, the `enhanced` model might perform better.

Remember that the quality of the audio data you're feeding into the model can greatly affect the transcription accuracy. The clearer the audio, the better the results you'll get.
> John Vajda
